src/schema.rs

```diff
@@ -650,6 +656,12 @@ diesel::table! {
         ///
         /// (Automatically generated by Diesel.)
         expires_at -> Nullable<Timestamp>,
+        /// The `action` column of the `publish_rate_overrides` table.
+        ///
+        /// Its SQL type is `Int4`.
+        ///
+        /// (Automatically generated by Diesel.)
+        action -> Int4,
```

Comment:
make sure to adjust the ops guide accordingly once this is merged 😉

Reply:
Yep, I'll do so once the second PR has been merged.